src/posthog-surveys-types.ts

```diff
@@ -6,8 +6,9 @@
 
 export interface SurveyAppearance {
     // keep in sync with frontend/src/types.ts -> SurveyAppearance
-    backgroundColor?: string
-    submitButtonColor?: string
+    backgroundColor: string
```

Comment:
Are you sure these will always exist, even on old surveys?

Reply:
fixed